bonsai/Bonsai.config

```diff
@@ -4,8 +4,9 @@
     <Package id="AForge" version="2.2.5" />
     <Package id="AForge.Video" version="2.2.5" />
     <Package id="AForge.Video.DirectShow" version="2.2.5" />
+    <Package id="AllenNeuralDynamics.LicketySplitLickDetector" version="0.1.0-preview2024020101" />
     <Package id="AssimpNet" version="4.1.0" />
-    <Package id="Bonsai" version="2.7.2" />
+    <Package id="Bonsai" version="2.8.0" />
```

Comment:
Do we have to manually install the updated bonsai version on each computer?

Reply:
No, we don't. It will be automatically updated when we open the bonsai (or the bonsai workflow).